chart/files/pod-template-file.kubernetes-helm-yaml

```diff
@@ -35,15 +35,16 @@ metadata:
     {{- if or (.Values.labels) (.Values.workers.labels) }}
       {{- mustMerge .Values.workers.labels .Values.labels | toYaml | nindent 4 }}
     {{- end }}
-  {{- if or .Values.airflowPodAnnotations .Values.workers.podAnnotations }}
   annotations:
-    {{- if .Values.airflowPodAnnotations }}
-      {{- toYaml .Values.airflowPodAnnotations | nindent 4 }}
-    {{- end }}
-    {{- if .Values.workers.podAnnotations }}
-      {{- toYaml .Values.workers.podAnnotations | nindent 4 }}
+    cluster-autoscaler.kubernetes.io/safe-to-evict: {{ .Values.workers.safeToEvict | quote}}
+    {{- if or .Values.airflowPodAnnotations .Values.workers.podAnnotations }}
+      {{- if .Values.airflowPodAnnotations }}
+        {{- toYaml .Values.airflowPodAnnotations | nindent 4 }}
+      {{- end }}
+      {{- if .Values.workers.podAnnotations }}
+        {{- toYaml .Values.workers.podAnnotations | nindent 4 }}
+      {{- end }}
```

Comment:
If `.Values.airflowPodAnnotations` and `.Values.workers.podAnnotations` have the same key with different values what will happen?

u can look in worker-deployment how it's been soved
https://github.com/apache/airflow/blob/65642ed9503f6135f781ac5e5158d375fb852d4a/chart/templates/workers/worker-deployment.yaml#L42